skopt/optimizer/optimizer.py

```diff
@@ -171,7 +170,7 @@ def __init__(self, dimensions, base_estimator="gp",
                  model_queue_size=None,
                  acq_func_kwargs=None,
                  acq_optimizer_kwargs=None):
-        self.specs = {"args": copy.copy(inspect.currentframe().f_locals),
+        self.specs = {"args": copy.deepcopy(locals()),
```

Comment:
I'm thinking, if you drop `self` from `locals()`, copying shouldn't be necessary any longer.

Reply:
Great call - didn't know `self` wasn't needed!